Appsilon/rhino
# Use `copy_template()`

### Changes
Closes #38:
1. Changed `R/init.R` to use `copy_template()` instead of `fs` package directly.
2. Template files renamed to utilize `dot.` and `.template` mechanism of `copy_template()`.
3. Templates structure unified so that each template is copied into the project root (necessary subdirectories are now part of the template itself).
3. CI workflow extension changed to `.yml` for consistency with `rhino.yml`.

### How to test
Initialize a project using `rhino::init()` from `develop` and from `use-copy-template` branch. Both projects should be identical (apart from `.yaml` -> `.yml` extension change for consistency).
